.github/workflows/nodejs.yml

```diff
@@ -18,3 +18,25 @@ jobs:
           yarn test
         env:
           CI: true
+      - name: Ensure that declarations changes have been committed
+        run: |
+          yarn declarations
+          git status # Exits if there are changes
```

Comment:
Can you explain or rephrase the comment please? I don't understand it.

Reply:
`git status` exists if there are changes.

Reply:
That does not make it better :D What do you mean with _exists_ here please?

I just tried on your branch: 

without changes: 
```
git status
On branch 220-breaking-api-change-detection
Your branch is up to date with 'origin/220-breaking-api-change-detection'.

nothing to commit, working tree clean
```

with changes: 
```
git status
On branch 220-breaking-api-change-detection
Your branch is up to date with 'origin/220-breaking-api-change-detection'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	declarations/ethereum_abi/

nothing added to commit but untracked files present (use "git add" to track)
```

Reply:
Check now :)